pgfinder/pgio.py

```diff
@@ -23,6 +24,21 @@
 LOGGER = logging.getLogger(LOGGER_NAME)
 
 
+def mass_libraries() -> Dict:
+    """
+    Loads built-in mass libraries from csv files as a dictionary.
+
+    `.csv` is chosen for consistency of data storage over the project.
+
+    Returns
+    -------
+    dict
+        A dictionary of filenames and paths to read from
```

Comment:
We can be a little more informative here.

```
A dictionary of mass libraries indexed by species name with paths to read them from as values.
```

Reply:
Improved the message a bit in the latest commit!